src/App.js

```diff
@@ -1,10 +1,19 @@
+import {Routes, Route} from 'react-router-dom';
+
+import Home from './components/Home';
 import Main from './components/Main';
+import Navigation from './components/Navigation';
+import Nutrition from './components/Nutrition';
 
 export default function App() {
   return (
-    <div>
-      <h2>Fitness App</h2>
-      <Main />
-    </div>
+    <>
+      <Home />
+      <Routes>
+        <Route path="/" element={<Navigation />} />
+        <Route path="/Main" element={<Main />} />
+        <Route path="/Nutrition" element={<Nutrition />} />
```

Comment:
I would recommend to use lowercase for the routes:
```suggestion
        <Route path="/main" element={<Main />} />
        <Route path="/nutrition" element={<Nutrition />} />
```